www/pages/working-with-qgis-plugin.md

```diff
@@ -98,6 +98,14 @@ You can create a new Mergin project using the plugin. The plugin offers two opti
 
 	**Note:** There should be only one QGIS project (qgs or qgz file) within the folder (or subfolders).
 
+The plugin allows you to create the project under your own name or an organization you are a member of, provided that you have write access to the name space.
+
+### Cloning an existing project
+
+With the plugin, you can make a copy of one of your existing projects or the ones shared with you. Simply right-click on the project under the **Browser panel > Mergin ** and select **Clone**. In the new window, select the **Owner** from the drop-down menu and type your new **Project Name**.

```

Comment:
it is possible to also clone public projects of others that were not shared with the user (e.g. people can use `lutraconsulting/Basic survey` project as a starting point)